tests/phpunit/TestFramework/PhpUnit/Coverage/IndexXmlCoverageReaderTest.php

```diff
@@ -33,23 +33,25 @@
 
 declare(strict_types=1);
 
-namespace Infection\Tests\TestFramework\Coverage\XmlReport;
+namespace Infection\Tests\TestFramework\PhpUnit\Coverage;
```

Comment:
We discussed it previously in many details, and I still think that coverage reader should not be in `TestFramework\PhpUnit` namespace, because it's used for all the test frameworks. So it's generic, not PHPUnit specific. 

Codeception generates xml report
PHPUnit
and PhpSpec does

Reply:
No prob, I'll move it a second.

Reply:
Right. The thing is I didn't move it in the first place. It is GitHub confuses it with `FileCodeCoverageProviderFactory` I removed. They were very similar, so this can be understood. Yet `IndexXmlCoverageReader` is exactly where `IndexXmlCoverageParser` is in the master.

Reply:
Please confirm. It is not a big deal to move around, but there should be no mistake if this is really required.

Reply:
You are right. I didn't notice that moment when `IndexXmlCoverageParser` was placed under `PhpUnit` namespace.

So the changes in this PR looks good to me.

PS: my concern is based on the following: if we ever happen to extract PHPUnit out of the core, these classes should not be extract together with the test framework, because they are part of the core and used for all adapters

Reply:
I cannot say I agree with their current location either. Shall we move them then?

I'll open a PR.

Reply:
👍 

Thank you!